scalableminds/webknossos
# Fix "Import Agglomerate Skeleton" tooltip

Ensure layer mappings are actually loaded if context menu tooltip is triggered to avoid showing the loading message forever. The same is done when hovering the proofreading tool button in the action bar.

See [this](https://scm.slack.com/archives/C5AKLAV0B/p1696854134360809) slack thread for context.

I also wanted to advertise (link to) https://webknossos.org/services/automated-segmentation in case no agglomerate or connectome file was found, but I didn't find a good solution. Linking in the tooltip is weird, because all our tooltip usually don't receive pointer events and are click-through. Changing that leads to tooltips in the context menu blocking other context menu entries. Adding a question mark with a link in the context menu entry itself also doesn't work since the disabled context menu entry is not clickable. Not sure how important this is and whether @hotzenklotz or @philippotto have any other ideas?

### URL of deployed dev instance (used for testing):
- https://fixagglomerateskeletontooltip.webknossos.xyz

### Steps to test:
- Open an annotation, open the context menu and hover over the "Import Agglomerate Skeleton" entry - It should not show "Checking for agglomerate files..." but a more specific reason for why it is disabled.

### Issues:
- fixes https://scm.slack.com/archives/C5AKLAV0B/p1696854134360809

------
(Please delete unneeded items, merge only when none are left open)
